src/python/pants/util/docutil.py

```diff
@@ -12,6 +12,10 @@ def terminal_width(*, fallback: int = 96, padding: int = 2) -> int:
     return shutil.get_terminal_size(fallback=(fallback, 24)).columns - padding
 
 
-def docs_url(slug: str) -> str:
-    """Link to the Pants docs using the current version of Pants."""
-    return f"https://www.pantsbuild.org/v{MAJOR_MINOR}/docs/{slug}"
+def bracketed_docs_url(slug: str) -> str:
```

Comment:
Could this be a boolean param? `slub:str, *, bracked: bool = True`.

Reply:
Then the function name would be misleading. If I saw a method called docs_url() I would assume it returned a URL. Plus  we don't need the URL right now anyway.